portfolio.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Fahad's world</title>
+    <link rel = "stylesheet" href="css/styles.css"
+</head>
+<body>
+        <header>
+                <p class = "pageHead"><a href = "index.html">About me  |  </a></p>

```

Comment:
When you head over to portfolio page the underline for the about me page extends more than it should. Can you guess the reason for this? Take a minute to think about it then read the answer. **Answer**: That because you've included the " | " inside the <a> tag when it should be in the <p> tag. So try heading to the portfolio page and see what happens when you move the " | " to outside the <a> tag.